config/package/manifests/resources/applicationintegration.aws.crossplane.io/queue.resource.yaml

```
@@ -0,0 +1,15 @@
+id: queue
+title: SQS Queue
+titlePlural: SQS Queues
+category: 
+overviewShort: "A Queue is a managed resource that represents an AWS SQS Queue."
+overview: |
+ A Queue is a managed resource that represents an AWS SQS Queue.
+readme: |
+ ## AWS SQS Queue
+
+ Amazon Simple Queue Service (SQS) is a fully managed message queuing service that enables you to decouple and scale microservices, distributed systems, and serverless applications. SQS eliminates the complexity and overhead associated with managing and operating message oriented middleware, and empowers developers to focus on differentiating work.
+
+ ---
+
+ You can learn more at <https://aws.amazon.com/sqs>.
```

```suggestion
 You can learn more at https://aws.amazon.com/sqs
```
Are the URLs in `<>` for a special kind of Markdown processing?

---

I was mostly following the existing format, but I believe this makes links be recognized